raytracing/thorlabs.py

```diff
@@ -96,6 +104,11 @@ def __init__(self):
 
 class AC254_125_A(AchromatDoubletLens):
     def __init__(self):
+        """ As of May 31 2020, the calculated edge thickness does not match
```

Comment:
Although I am convinced it is an error, I am unable to confirm this number.  Let's leave it like that for now.